crates/sui-core/src/consensus_handler.rs

```diff
@@ -230,6 +239,37 @@ impl<T: ParentSync + Send + Sync> ExecutionState for ConsensusHandler<T> {
     }
 }
 
+struct AsyncTransactionScheduler {
+    sender: tokio::sync::mpsc::Sender<Vec<VerifiedExecutableTransaction>>,
+}
+
+impl AsyncTransactionScheduler {
+    pub fn start(
+        transaction_manager: Arc<TransactionManager>,
+        epoch_store: Arc<AuthorityPerEpochStore>,
+    ) -> Self {
+        let (sender, recv) = tokio::sync::mpsc::channel(16);
```

Comment:
Can we have some visibility on if this becomes a bottleneck?

Reply:
Sure, I can add utilization timer